src/control/client/storage.go

```diff
@@ -154,7 +154,8 @@ func (c *connList) ListStorage() (ClientNvmeMap, ClientScmMap) {
 }
 
 func (c *control) formatStorage() error {
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	// Maximum time limit for format is 2hrs
+	ctx, cancel := context.WithTimeout(context.Background(), 120*time.Minute)
```

Comment:
Just curious, this seems like a very long time, though plausible for formatting a lot of SSDs. Based on empirical testing, or just a guess?

Reply:
This sets a maximum time out to cover worst-case scenarios (16 SSDs that have to be formatted sequentially through SPDK). in most cases request will return within a few minutes but time can vary due to firmware version, model and extent of usage

Reply:
Would it be possible to format them in parallel rather than sequentially? Not something I'd necessarily expect changed in this PR, but might make for a better user experience if it can take a very long time.

Reply:
I would like to but I'm not sure it's possible, I'm going to investigate this is possible through SPDK